src/common/login/PostLoginActions.ts

```diff
@@ -50,6 +50,7 @@ export class PostLoginActions implements PostLoginAction {
 		private readonly userManagementFacade: UserManagementFacade,
 		private readonly customerFacade: CustomerFacade,
 		private readonly showSetupWizard: () => unknown,
+		private readonly appPartialLoginSuccessActions: () => unknown,
```

Comment:
I recommend you just make another call to `logins.addPostLoginAction()` in your appropriate locator instead of adding it here

Reply:
I'm afraid that's not true

```ts
class LoginController {
  // ...
    addPostLoginAction(handler: lazy<Promise<PostLoginAction>>) {
        this.postLoginActions.push(handler)
    }
  // ...
}
```

```ts
export interface PostLoginAction {
	/** Partial login is achieved with getting the user, can happen offline. The login will wait for the returned promise. */
	onPartialLoginSuccess(loggedInEvent: LoggedInEvent): Promise<void>

	/** Full login is achieved with getting group keys. Can do service calls from this point on. */
	onFullLoginSuccess(loggedInEvent: LoggedInEvent): Promise<void>
}
```

you can add another `PostLoginAction` in your locator that runs has `onPartialLoginSuccess()`